lib/classes/Swift/Transport/AbstractSmtpTransport.php

```diff
@@ -311,18 +317,28 @@ public function getBuffer()
      * @param string   $command
      * @param int[]    $codes
      * @param string[] $failures An array of failures by-reference
+     * @param bool     $pipeline Do not wait for response
      *
-     * @return string
+     * @return string|null The server response, or null if pipelining is enabled
      */
-    public function executeCommand($command, $codes = [], &$failures = null)
+    public function executeCommand($command, $codes = [], &$failures = null, $pipeline = false)
     {
         $failures = (array) $failures;
         $seq = $this->buffer->write($command);
-        $response = $this->getFullResponse($seq);
         if ($evt = $this->eventDispatcher->createCommandEvent($this, $command, $codes)) {
             $this->eventDispatcher->dispatchEvent($evt, 'commandSent');
         }
-        $this->assertResponseCode($response, $codes);
+
+        $this->pipeline[] = [$command, $seq, $codes];
+        if ($pipeline && $this->pipelining) {
```

Comment:
Is there a case where you would want to not use pipelining if the SMTP supports it?

Reply:
Hmm, there are some (old) reports about compatibility issues:
* http://postfix.1071664.n5.nabble.com/Disable-outbound-smtp-pipelining-td40604.html
* ftp://ftp.cs.uu.nl/mirror/postfix/workarounds.html

I have added a method that allows forcing pipelining to be enabled or disabled.